packages/picker/test/index.ts

```diff
@@ -839,24 +839,35 @@ export function runPickerTests(): void {
             ) as MenuItem;
 
             const opened = oneEvent(el, 'sp-opened');
-            el.click();
+
+            const boundingRect = el.button.getBoundingClientRect();
+            sendMouse({
+                steps: [
+                    {
+                        type: 'click',
+                        position: [
+                            boundingRect.x + boundingRect.width / 2,
+                            boundingRect.y + boundingRect.height / 2,
+                        ],
+                    },
+                ],
+            });
```

Comment:
Updated this from `el.click()` to coordinate-based mouse event simulation which now properly triggers the event propagation chain and more accurately mimics real user interaction